### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by musangowope**
- Created services file to initialize localization using i18next 
- Created parse config to scan through project and extract translations from file (i.e look for instances of t function) and generate translation json files
- Added translations syntax to all relevant Pages and components and generated english json translation files
- Created Language Switcher
- finalize json import logic for inlining resources in service file i18next service file
- create corresponding sample language translation jsons to demo language switching (Did so for globals.json) translating to zulu

* [x] change constitution language when the app's language changes

### CODE REVIEW

File: src/pages/help/Help.tsx [hunk 1]
```diff
@@ -3,45 +3,49 @@ import { IonCol, IonContent, IonIcon, IonPage } from "@ionic/react";
...
-          <h2>About</h2>
+          <h2>{t('about_heading', 'About')}</h2>
         </section>
 
         <hr className="header-divider" />
-        <p>
-          The Constitution App is developed by the{" "}
+        {parse(t('app_description', `
+          <p>
+            The Constitution App is developed by the&nbsp;
```
Comment (longhotsummer) [inline src/pages/help/Help.tsx:24]: I'm not sure why the `{" "}` was there in the first place, but I'm also not sure that they need to be replaced with non-breaking spaces. Is a normal space suitable?

Comment (longhotsummer) [reply src/pages/help/Help.tsx:24]: Normally we'd do that so it doesn't line wrap at that point, but I don't think it matters?

File: src/pages/help/Help.tsx [hunk 2]
```diff
@@ -53,39 +57,43 @@ const Tab1: React.FC = () => {
...
+              https://creativecommons.org/licenses/by-nc/4.0/
+            </a>
+            .
+          </p>
+        `))}
         <div>
           <ActionAnchorLink
             href="mailto:info@africanlii.org"
-            actionText="Send feedback"
+            actionText={parse(t('send_feedback_button_title', 'Send feedback'))}
```
Comment (longhotsummer) [inline src/pages/help/Help.tsx:96]: is the parse() necessary here?

File: src/services/i18next.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+const enResources = {};
+Object.keys(enData).forEach(ns => {
+  if(enData[ns]) {
+    enResources[ns] = enData[ns];
+  }
+})
+
+const zuData = {
+  case: loadJSONFile("en/case.json"),
```
Comment (longhotsummer) [inline src/services/i18next.js:32]: `zu/case.json`?

File: src/pages/help/Help.tsx (hunk 1 above)
Comment (musangowope) [reply src/pages/help/Help.tsx:24]: > 

yea lets remove the nbsp. its not actually needed

Comment (musangowope) [reply src/pages/help/Help.tsx]: yea, nbsp or `{" "}`is not necessary for these `p` tags

File: src/pages/help/Help.tsx (hunk 2 above)
Comment (musangowope) [reply src/pages/help/Help.tsx]: nope its not. refactored

File: src/services/i18next.js (hunk 3 above)
Comment (musangowope) [reply src/services/i18next.js]: made a cleaner implementation

**Review by longhotsummer — APPROVED** [review]

**Review by longhotsummer — APPROVED** [review]

**Comment by longhotsummer** [comment]
@musangowope when you have a moment, please add `package-lock.json` to this PR. It's not in the repo and I'm getting some weird React errors, and I think I'm on a different version of things to you.

**Comment by musangowope** [comment]
@longhotsummer done

> @musangowope when you have a moment, please add `package-lock.json` to this PR. It's not in the repo and I'm getting some weird React errors, and I think I'm on a different version of things to you.
done

**Pull request merged**